src/templates/person.html

```diff
@@ -152,6 +152,59 @@ <h2>{{ person.email }}</h2>
         </form>
       </div>
     </div>
+    <!-- Postbox -->
+    <div class="panel panel-default">
+      <div class="panel-heading"><h4>Create Postbox item</h4></div>
+      <div class="panel-body">
+        <form method="POST" action="/__BACKOFFICE__/createPostboxItem/{{ person.id }}">
+          <div class="form-group">
+            <label for="ownerType">Owner type</label>
+            <select name="ownerType" class="form-control" required>
+              <option {% if person.customer_vetting_status === "Person" %}selected {% endif %} value="Person">Person</option>
+              <option {% if person.customer_vetting_status === "Business" %}selected {% endif %} value="Business">Business</option>
+            </select>
+          </div>
+          <div class="form-group">
+            <label for="name">Document name</label>
+            <input class="form-control" type="text" name="name" placeholder="Document name" autocomplete="off" required />
+          </div>
+          <div class="form-group">
+            <label for="description">Document description</label>
+            <input class="form-control" type="text" name="description" placeholder="Document description" autocomplete="off" required />
+          </div>
+          <div class="form-group">
+            <label for="documentType">Document type</label>
+            <select name="documentType" class="form-control" required>
+              <option {% if person.customer_vetting_status === "ACCOUNT_STATEMENT" %}selected {% endif %} value="ACCOUNT_STATEMENT">ACCOUNT_STATEMENT</option>
```

Comment:
nit: you could pass enum to handler and then iterate on it here to create option elements

Reply:
is it for sure `customer_vetting_status`?

Reply:
removed :+1: